plugins/smart_proxy_dynflow/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-smart-proxy-dynflow (0.8.11-1) stable; urgency=low
```

Comment:
This package is for the smart_proxy_dynflow plugin, but that's version 0.0.7 or 0.1.x. This file isn't for dynflow itself - you updated that in `dependencies/*/dynflow`.